client/state/reader/tags/items/actions.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { memoize, trim } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_FETCH_TAG_REQUEST,
+	READER_FETCH_TAG_RECEIVE,
+	READER_FETCH_TAGS_REQUEST,
+	READER_FETCH_TAGS_RECEIVE,
+	READER_FOLLOW_TAG_REQUEST,
+	READER_FOLLOW_TAG_RECEIVE,
+	READER_UNFOLLOW_TAG_REQUEST,
+	READER_UNFOLLOW_TAG_RECEIVE,
+} from 'state/action-types';
+
+/**
+ * Helper function. Turns a tag name into a tag "slug" for use with the API.
+ *
+ * @param  {String} tag  Tag name to parse into a slug
+ * @return {String}      Tag slug
+ */
+const unmemoizedSlugify = ( tag ) => encodeURIComponent(
```

Comment:
looks like it would work pretty well. tested with emoji and non-latin characters.